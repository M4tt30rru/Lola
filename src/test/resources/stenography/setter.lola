##Import "std.lola"
##Find ##Any(prot) <- ##Any(mods) ##Type(t) ##Identifier(n);
 ##run {
setter = 'set' + n.name[0].upper() + n.name[1:]
 }
 ##replace ##(mods) ##(t) ##(n); ##(prot) void ##(setter) (##(t) __) {this.##(n) = __;}

protected <- private Integer i;
public <- protected transient String str;
