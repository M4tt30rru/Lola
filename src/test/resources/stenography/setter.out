protected transient String str;
private Integer i;
protected void setI(Integer __) {
 this.i = __;
}
public void setStr(String __) {
 this.str = __;
}
